CLI/Analyze.cpp

```diff
@@ -236,9 +243,9 @@ int main(int argc, char** argv)
         if (isDirectory(argv[i]))
         {
             traverseDirectory(argv[i], [&](const std::string& name) {
-                if (name.length() > 4 && name.rfind(".lua") == name.length() - 4)
-                    failed += !analyzeFile(frontend, name.c_str(), format, annotate);
-            });
+                    if (name.length() > 4 && name.rfind(".lua") == name.length() - 4)
```

Comment:
This whitespace change should probably be reverted, otherwise this change will conflict with #124

Reply:
Guess I committed part of clang-format's changes. It looks like it's trying to expand the lambda further.